_layouts/default.html

```diff
@@ -209,7 +209,7 @@ <h3>Why Another Workflow Engine?</h3>
 						Cylc is built to transparently use a pool of host VMs, with
 						load balancing at start-up, and it scales arbitrarily well with the
 						number of host VMs. Running workflows can self-migrate to another
-						host at maintenance time. </p> 
+						host at maintenance time. </p>
```

Comment:
Changes above were added automatically by the IDE, but I thought it was OK. Removing extra spaces.